libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -111,175 +115,178 @@ export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
-            </Stack.Item>
-            <Stack
-              tokens={{ childrenGap: "l2" }}
-              className={classNames.sectionIndent}
-            >
+            <Stack>
+              <Stack.Item>
+                <Separator className={classNames.separator} />
+              </Stack.Item>
               <Stack.Item>
                 <Text variant="large" className={classNames.title}>
-                  {localization.InterpretVision.Dashboard.panelInformation}
+                  {localization.InterpretVision.Dashboard.panelExplanation}
                 </Text>
               </Stack.Item>
-              <Stack.Item className={classNames.featureListContainer}>
-                <List
-                  items={this.state.metadata}
-                  onRenderCell={this.onRenderCell}
-                />
-              </Stack.Item>
+              <Stack>
+                {
+                  <ComboBox
+                    id={localization.InterpretVision.Dashboard.objectSelect}
+                    label={localization.InterpretVision.Dashboard.chooseObject}
+                    onChange={this.selectODChoiceFromDropdown}
+                    selectedKey={this.state.odSelectedKey}
+                    options={this.state.selectableObjectIndexes}
+                    className={"classNames.dropdown"}
+                    styles={FluentUIStyles.smallDropdownStyle}
+                  />
+                }
+                <Stack>
+                  {!this.props.loadingExplanation[item.index][
+                    +this.state.odSelectedKey.slice(7)
+                  ] ? (
+                    <Stack.Item>
+                      <Image
+                        src={`data:image/jpg;base64,${this.props.explanations
+                          .get(item.index)
+                          ?.get(+this.state.odSelectedKey.slice(7))}`}
+                        width={explanationImageWidth}
+                        style={explanationImage}
+                      />
+                    </Stack.Item>
+                  ) : (
+                    <Stack.Item>
+                      <Spinner
+                        label={`${localization.InterpretVision.Dashboard.loading} ${item?.index}`}
+                      />
+                    </Stack.Item>
+                  )}
+                </Stack>
+              </Stack>
             </Stack>
           </Stack>
         </Panel>
       </FocusZone>
     );
   }
 
-  private onRenderCell = (
-    item?: Array<string | number | boolean> | undefined
-  ): React.ReactNode => {
-    if (!item) {
-      return;
+  private selectODChoiceFromDropdown = (
+    _event: React.FormEvent<IComboBox>,
+    item?: IComboBoxOption
+  ): void => {
+    if (typeof item?.key === "string") {
+      this.setState({ odSelectedKey: item?.key });
+      if (this.state.item !== undefined) {
+        // Remove "Object: " from labels. We only want index
+        this.props.onChange(this.state.item, +item.key.slice(ExcessLabelLen));
```

Comment:
since "Object:" might subject to change in the future, ExcessLabelLen might change. Can we define const ExcessLabelLen as length of the prefix, instead of the hardcoded number? It's easier for maintaining.